content/applications/general/calendars/outlook/outlook_calendar.rst

```diff
@@ -1,103 +1,149 @@
-     the **first** sync, Outlook will also send confirmation emails of all scheduled events
-     (including past ones).
-   | To avoid this situation, we recommend *archiving all past events, and removing yourself from
-     future ones in Odoo Calendar before the first sync*. This will prevent events from being
-     created in Outlook, and therefore, no redundant emails will be sent during the first
-     synchronization. After the sync, events will only send one confirmation email.
+Warnings (before sync)
+----------------------
+
+Odoo highly recommends testing the Outlook calendar synchronization on a test database and a test
+email address (that is not used for any other purpose) before attempting to sync the desired
+Outlook Calendar with the user’s production database.
+
+If the user has any past, present, or future events on their Odoo calendar before syncing their
+Outlook calendar, Outlook will treat the events pulled from Odoo's calendar during the sync as new
+events, causing an email notification to be sent from Outlook to all the event attendees.
+
+To avoid unwanted emails being sent to all past, present, and future event attendees, the user must
+add the events from the Odoo calendar to the Outlook calendar before the first ever sync, delete
+the events from Odoo, and then start the sync. 
+
+Even after synchronizing the Odoo Calendar with the Outlook calendar, Outlook will still send a
+notification to all event participants every time an event is edited (created, deleted, unarchived,
+or event date/time changed), with no exceptions. This is a limitation that cannot be fixed from
+Odoo’s side.
+
+After one user syncs their Outlook calendar to the Odoo database, unwanted email notifications are
+unavoidable because the first synchronized user’s events will be in the Odoo Calendar. If the Odoo
+database is shared amongst multiple users, and another user wants to sync their Outlook calendar
+with Odoo Calendar, Outlook will again pull the existing Odoo Calendar events during the sync and
+treat them as new events, causing Outlook to send email invitations to all event attendees.
+
+In summary, once a user synchronizes their Outlook calendar with the Odoo calendar:
+
+- Creating an event in Odoo causes Outlook to send an invitation to all event attendees.
+- Deleting an event in Odoo causes Outlook to send a cancellation to all event attendees.
+- Unarchiving an event in Odoo causes Outlook to send an invitation to all event attendees.
+- Archiving an event in Odoo causes Outlook to send a cancellation to all event attendees.
+- Adding a contact to an event causes Outlook to send an invitation to all event attendees.
+- Removing a contact from an event causes Outlook to send a cancellation to all event attendees.
+
+If the user decides to synchronize, Odoo suggests that the user sets the maximum allowed expiration
+date for the Client Secret in Microsoft Azure’s portal, so there is no need to re-synchronize soon.
```

Comment:
Note: lines 117-121 sounds like a wordier version of what's already written on line 49-51, however some of the language is identical. 

Not sure if that was intentional or written again for emphasis?